onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2579,6 +2579,10 @@ ORT_API_STATUS_IMPL(OrtApis::Graph_GetName, _In_ const OrtGraph* graph, _Outptr_
   API_IMPL_END
 }
 
+ORT_API(const ORTCHAR_T*, OrtApis::Graph_GetModelPath, _In_ const OrtGraph* graph) {
+  return graph->GetModelPath();
```

Comment:
>return graph->GetModelPath() [](http://example.com/codeflow?start=2&length=28)

I would still prefer exception guards here. And with that it would raise a question of how to report potential error translated from the exception.

Even standard types such as std::path can throw.

Reply:
Updated